index.html

```diff
@@ -10,234 +10,252 @@
-    <div class="menuSection">
-      <h1 class="introduction">I’m Mariane Glad to see you!</h1>
-      <p class="Whoiam">
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact
-        me.
-      </p>
-    </div>
-    <div class="menuSection">
-      <a href="#" target="_blank">LET'S CONNECT</a>
-      <ul class="socialMedia">
-        <li>
-          <img src="image/Twitter.png" alt="" />
+  <div class="container">
+    <header>
+      <nav class="navmenu">
+        <h3>My Logo</h3>
+      </nav>
+      <img class="onion" src="image/Union.png" alt="" />
+      <div class="rest">
+        <div>Portfolio</div>
+        <div>About</div>
+        <div>Contact</div>
+      </div>
+    </header>
+    <section class="backgroundimg">
+      <div class="menuSection menuSection1">
+        <h2 class="introduction">I’m Mariane Glad to see you!</h2>
+        <p class="Whoiam">
+          I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+      </div>
+      <div class="menuSection">
+        <a href="#" target="_blank">LET'S CONNECT</a>
+        <ul class="socialMedia">
+          <li>
+            <img src="image/Twitter.png" alt="" />
+          </li>
+          <li>
+            <img src="image/Linkedin.png" alt="" />
+          </li>
+          <li>
+            <img src="image/Meduim.png" alt="" />
+          </li>
+          <li>
+            <img src="image/Github.png" alt="" />
+          </li>
+          <li>
+            <img src="image/angellist.png" alt="" />
+          </li>
+        </ul>
+      </div>
+    </section>
+    <div id="cards">
+      <div id="flex">
+        <div class="card">
+          <img class="img" src="image/card-1.svg" alt="" />
+          <div class="content">
+            <div class="topic">Tonic</div>
+            <div class="sub-topic">
+              <span>CANOPY</span>
+              <span class="dot"></span>
 
-        </li>
-        <li>
-          <img src="image/Linkedin.png" alt="" />
+              <span>Back End Dev</span>
+              <span class="dot"></span>
 
-        </li>
-        <li>
-          <img src="image/Meduim.png" alt="" />
+              <span>2015</span>
+            </div>
+            <div class="description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </div>
+            <div class="button-links">
+              <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+            </div>
+            <a href="#" class="project">See Project</a>
```

Comment:
## KIndly Fix that for all the cards 💯💯

- [ ] I think here you need to put a margin between the header section and between the cards so kindly add it to match the Figma design as much as possible. ✅✅✅

- [ ] As I can see the background color for all the cards is different than the Figma so please change it to be the same 💪🏻💪🏻

- [ ] I noticed the size of all the cards is greater in your design so kindly decrease the width to match the Figma 

- [ ] Kindly make some spaces margin/padding between the cards' information ✅💪🏻


- Your design
![image](https://user-images.githubusercontent.com/58553711/179356020-894157cf-7cf9-4178-a168-e9c8611dca25.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/179356085-5f4a818d-7bf1-498f-89cd-6904278db658.png)